crates/zkevm_test_harness/src/witness/individual_circuits/events_sort_dedup.rs

```diff
@@ -93,12 +93,12 @@ pub(crate) fn compute_events_dedup_and_sort<
         .simulator
         .witness
         .iter()
-        .map(|el| &el.0)
+        .map(|el| el.0)
         .collect();
     let rhs_contributions: Vec<_> = intermediate_sorted_simulator
         .witness
         .iter()
-        .map(|el| &el.0)
```

Comment:
This change doesn’t affect behavior: it is copying `.0` now as well. 

I can think about more radical change, so as not to copy at all, but it is better to do as in RAM permutation - just collect in in a separate vector from the start and move into corresponding function.

Basically this circuit loads RAM **much** less than RAM permutation, so I did not focus on changing it.

Reply:
Sorry, yes, you are right. I confused it with some other cases.

This change is needed to avoid creating a vector of pointers in the RAM permutation witness generation, but to directly use the values ​​themselves. This saves a lot of memory, much more than is used here.